manifests/cf-manifest/manifest/020-cf-properties.yml

```diff
@@ -148,6 +148,7 @@ properties:
     extra_headers_to_log:
     debug_addr:
     enable_routing_api:
+    drain_wait: 15
```

Comment:
Can you explain better in the commit message why this happens:

> Set gorouter drain_wait to 15s 
>
> With this setting the gorouter will keep serving to new connections for 15 seconds after getting the drain signal, but respond 500 error when the HTTPMonitor user agent is used. 
>
> As the ELBs  healthcheck runs every 5 seconds and fails after 2 failures, and it will send the custom header (currently by using a custom frontent in haproxy). 
>
> This way we ensure that we don't drop any incoming connection before the ELB marks the gorouter down.